zellij-utils/src/input/theme.rs

```diff
@@ -48,6 +51,71 @@ pub struct PaletteFromYaml {
 pub enum PaletteColorFromYaml {
     Rgb((u8, u8, u8)),
     EightBit(u8),
+    Hex(HexColor),
+}
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
+pub struct HexColor(u8, u8, u8);
+
+impl From<HexColor> for (u8, u8, u8) {
+    fn from(e: HexColor) -> (u8, u8, u8) {
+        let HexColor(r, g, b) = e;
+        (r, g, b)
+    }
+}
+
+pub struct HexColorVisitor();
+
+impl<'de> Visitor<'de> for HexColorVisitor {
+    type Value = HexColor;
+
+    fn expecting(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
+        write!(formatter, "a hex color in the format #RGB or #RRGGBB")
+    }
+
+    fn visit_str<E>(self, s: &str) -> Result<Self::Value, E>
+    where
+        E: Error,
+    {
+        if s.starts_with("#") {
+            return self.visit_str(&s[1..]);
+        }
```

Comment:
Hey, almost done!

Could you please apply clippy's recommendations as follows?

```rust
if let Some(stripped) = s.strip_prefix('#') {
    return self.visit_str(stripped);
}
```

Reply:
done